config/fs.go

```diff
@@ -58,6 +63,8 @@ type FSConfig struct {
 	FuseConfig `toml:"fuse"`
 
 	BackgroundFetchConfig `toml:"background_fetch"`
+
+	ContentStoreConfig store.Config `toml:"content_store"`
```

Comment:
Should this be moved to the root config?

Reply:
That's a good question. On its own merits, I decided not, but the consolidation of the other configs happened while I was working on this. Did we make plans for future configs to also be consolidated?

Reply:
Soliciting input from @Kern--

Reply:
Since we have a [metadata config](https://github.com/awslabs/soci-snapshotter/blob/main/config/config.go#L74) in the root I thought it might make sense to move this there as well. Nonetheless, I don't have a strong preference and I don't think its a blocker for merging.

Reply:
The consolitdation is in memory only. Was it an accident? We embed each subconfig so they end up appearing on the root config, but we still have the hierarchy in toml.

I think the default content store is a property of the root config. If we were configuring the content store itself, then I think it makes sense as a sub key in toml:

e.g.
```
content_store = "containerd"

[contentstore.containerd]
    address = "..."
```

thoughts?

Reply:
I'm wary of adding top level options, but we don't have a standard for where to put config so I'm not entirely opposed. It does make the parsing a little more complex.

Reply:
Will consolidate source file and package configs per discussion.
Will not change config struct.